app/views/pages/login.slim

```diff
@@ -0,0 +1,15 @@
+- if !cookies['return_to'] 
+	head
+		meta http-equiv="refresh" content="0; URL='/'"
```

Comment:
What's the purpose of this redirect? We may not be explicitly sending people here without a `return_to` cookie, but afaict we can still handle them fine if they happen to arrive without one for whatever reason. I personally sometimes type sign-in URLs directly into my address bar, or bookmark them.

Reply:
I hear what you're saying, and I agree that it makes more sense for the page to be accessible even if the user isn't currently authorizing an application. Should we still redirect a user who's already signed in, or display some other content?

Reply:
Sure, signing in while signed in is unexpected so we can redirect if that's the case. Instead of a meta tag though, the controller tends to be the best place for redirects. In this case `app/controllers/pages_controller.rb` would be changed to something like
```ruby
class PagesController < ApplicationController
  def login # or whatever the "to: pages#login" part ends up being in config/routes.rb
    redirect_to(root_path) if current_user.present?
  end
end
```